### DISCUSSION TITLE
Getting npm error in Build plugins collectionEditor Jenkins Job

### DISCUSSION THREAD

**Discussion by nikhilvarma940** [Sunbird-Ed/Community#492] [Installation] [answered]
Hi Team,

Getting npm error in Build plugins CollectionEditor please help me to resolve this issue

<img width="1655" alt="Screenshot 2023-05-15 at 7 10 47 PM" src="https://github.com/Sunbird-Ed/Community/assets/63706239/48d57f27-b6f3-4b51-a7df-4461307833fb">

**Comment by vinukumar-vs** [comment]
@nikhilvarma940 I would suggest to have this discussion thread in [Knowlg BB discussions](https://github.com/orgs/Sunbird-Knowlg/discussions) as they are the owning this module. It will also help community to find related issues at one place. 

[@pallakartheekreddy](https://github.com/pallakartheekreddy) [@HarishGangula](https://github.com/HarishGangula) Can you please help

**Reply by Krishnaj20** [reply]
@pallakartheekreddy @harishgangula - Any update on this please?

**Reply by HarishGangula** [reply]
@nikhilvarma940  Looks like that is old version of the collection editor, all the latest versions are available as npm packages please check https://www.npmjs.com/package/@project-sunbird/sunbird-collection-editor

**Reply by gandham-santhosh** [reply]
@HarishGangula this is a CollectionEditor build job. There is another thread for the same https://github.com/orgs/Sunbird-Ed/discussions/496 
It was working earlier but failing now. Is it failing due to deprecation of any public npm packages ? Should we need to change the build Github TAG?

**Reply by HarishGangula** [reply]
@gandham-santhosh  Collection editor changed to npm package and we are publishing it, so this job is not required to use collection editor now.

**Reply by gandham-santhosh** [reply]
Okay, I understand. But so far for all Sunbird Ed releases it is leveraging the Collection Editor as a custom build, not as a package.  How can this issue be fixed now considering the way it is being used in Ed?
@vinukumar-vs are you aware of this? is this in Ed upcoming development plans?

**Reply by gandham-santhosh** [reply]
@HarishGangula @pallakartheekreddy can you please help here

**Reply by vinukumar-vs** [reply, marked as answer]
@gandham-santhosh Collection-Edtior(v2) is packaged as NPM module of protal. 
https://github.com/Sunbird-Ed/SunbirdEd-portal/blob/f5a75aa2a222a7af48fee7157f65f1f13a71a043/src/app/client/package.json#L68

@rajeevsathish

**Reply by chitranshu-keshav** [reply]
@gandham-santhosh @rajeevsathish is this issue resolved? can you please confirm and update this thread
